enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -1,73 +1,88 @@
+    <meta name="viewport" content="user-scalable = no, width=device-width, initial-scale=1"/>
+    <script type="text/javascript" src="${static.url('common/js/vendor/jquery.js')}"></script>
+    <script type="text/javascript" src="${static.url('js/vendor/jquery.cookie.js')}"></script>
+    <script type="text/javascript" src="${static.url('enterprise/grant_data_sharing_permissions.js')}"></script>
+  </head>
+  <body>
+    <header class="logo-container" aria-label="${ title_bar_prefix }">
+      <h1>
+        <img src="${static.url('images/logo.png')}" alt="${ platform_name }"/>
+      </h1>
+    </header>
+    <hr class="pinstripe"/>
+    <main>
+      <div class="consent-container">
+        <h2 class="consent-title">${ consent_message_header }</h2>
+        <div class="consent-message">
+          <p>${ consent_request_prompt }</p>
+          <p>${ requested_permissions_header }
+          <ul class="consent-items">
+            % for permission in requested_permissions:
+            <li>${ permission }</li>
+            % endfor
+          </ul>
+          </p>
+        </div>
+        <div class="consent-input-container login-register">
+          <p class="agreement-text">${agreement_text}</p>
+          <form name="data-sharing-consent" method="POST" id="data-sharing">
+            <button type="submit" class="background-input consent-agreement-button" id="consent-button" name="data_sharing_consent" value="True">${ continue_text }</button>
+            <button type="button" class="failure-link background-input" id="failure-link">${ abort_text }</a>
+            <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }" />
+            % if course_specific and course_id:
+            <input type="hidden" name="course_id" value="${ course_id }" />
+            % endif
+            % if redirect_url:
+            <input type="hidden" name="redirect_url" value="${redirect_url}" />
+            % endif
+            % if failure_url:
+            <input type="hidden" name="failure_url" value="${failure_url}" />
+            % endif
+            % if enrollment_deferred:
+            <input type="hidden" name="enrollment_deferred" value="true" />
+            % endif
+          </form>
+        </div>
+        <br/>
+        <button type="button" id="consent-policy-dropdown-bar" class="consent-policy-dropdown-bar background-input" aria-controls="consent-policy">
+          <span class="dropdown-text">${ policy_dropdown_header }</span>
+          <span class="fa-stack fa-lg dropdown-icon-container" aria-hidden="true">
+            <span id="consent-policy-dropdown-icon-surround" class="fa fa-circle fa-stack-2x" aria-hidden="true"></span>
+            <span id="consent-policy-dropdown-icon" class="fa fa-chevron-right fa-stack-1x" aria-hidden="true"></span>
+          </span>
+        </button>
+        <div class="consent-policy" id="consent-policy" style="display:none;" aria-expanded="false">
```

Comment:
the ```aria-expanded``` attr needs to go on the button that controls this region, not the region itself.  I know its confusing, but it is intended to describe the state of the region it controls.